hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerMetrics.java

```diff
@@ -37,27 +36,16 @@
 
   private final MetricsSystem ms;
 
-  @Metric(about = "The total amount of used space in GigaBytes that needs to " +
-      "be balanced.")
-  private MutableGaugeLong dataSizeToBalanceGB;
+  @Metric(about = "The amount of Gigabytes that Container Balancer moved" +
+      " in the last iteration.")
+  private MutableCounterLong dataSizeMovedGB;
 
-  @Metric(about = "The amount of Giga Bytes that have been moved to achieve " +
-      "balance.")
-  private MutableGaugeLong dataSizeMovedGB;
+  @Metric(about = "Number of containers that Container Balancer moved" +
+      " in the last iteration.")
+  private MutableCounterLong movedContainersNum;
```

Comment:
Sure, I'll update the names. Aggregate metrics - you mean total no. of containers moved across iterations etc? For that we'll need to save container ids across iterations to avoid counting the same container multiple times.

Reply:
As we discussed we can have a simple metric. If a container is moved multiple times we can still count those moves. But this can be done in a separate jira.